.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
```

Comment:
Could we use the template from https://github.com/alphagov/.github/blob/master/workflow-templates/digitalmarketplace-python-ci.yml ? We have updated the cache key and restore logic.

Reply:
Ah, good point

Reply:
Done